### ISSUE TITLE
Support `aria-description`, `aria-describedby`

### ISSUE THREAD

**Issue by fendiaoxiaoshuazi** [Esri/calcite-design-system#13272]
**monday.com sync:** #10513380947

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Which Component

Input Date Picker, Input Time Picker, Slider

### Actual Behavior

In Survey123, we allow survey author to add question label and question description, and these information should be announced by the screen reader like NVDA. The current Label component cannot satisfy our needs, and we will need `aria-description`, `aria-describedby` (at least `aria-description`) to make the survey form 508 compliant. 

### Expected Behavior

Support `aria-description`, `aria-describedby`

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/label/ 

### Reproduction Steps

NA

### Reproduction Version

3.3.3

### Working W3C Example/Tutorial

https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA7 https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA1

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [x] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Survey123

**Comment by geospatialem** [comment]
The `input-date-picker` and `input-time-picker` support an `aria-label` via the `label` prop as you mention above. 

@fendiaoxiaoshuazi Can you dive into the use case more of why its not currently fitting the use case in Survey123? Once a user adds the context the component could be fed the value to satisfy the `aria-label` need.

**Comment by fendiaoxiaoshuazi** [comment]
Thanks @geospatialem , we tried the label prop, and it seems to work for us. I have updated the issue to support `aria-description` `aria-describedby`

**Comment by RuiminLi98** [comment]
Hi, my name is Ruimin Li from Arcgis Enterprise team, we have a similar issue (https://devtopia.esri.com/WebGIS/arcgis-portal-app/issues/70330) that need to set `aria-describedBy` to `calcite-input`. Thank you

**Comment by geospatialem** [comment]
> Hi, my name is Ruimin Li from Arcgis Enterprise team, we have a similar issue (https://devtopia.esri.com/WebGIS/arcgis-portal-app/issues/70330) that need to set `aria-describedBy` to `calcite-input`. Thank you

@RuiminLi98 Hi Ruimin, you mentioned `input`, but the request above is scoped to `input-date-picker`, `input-time-picker` and `slider`. Can you speak more to the use of the describedby attribute? 

Instead, would recommend coupling the component with Input Message, similar to one of our component samples instead: https://codepen.io/geospatialem/pen/MYpmbaK to accommodate more users. Would that work with the use case?

**Comment by RuiminLi98** [comment]
Hi @geospatialem , I was asked to left a comment about my current issue under this ticket. (https://teams.microsoft.com/l/message/19:fd15b51dacd24e70895ec1218a54ae06@thread.skype/1790121756791?tenantId=aee6e3c9-711e-4c7c-bd27-04f2307db20d&groupId=56fae21a-9407-4943-859f-a9bfcf0bbad3&parentMessageId=1789777424981&teamName=Calcite%20Design%20System&channelName=Get%20Help%20-%20Development&createdTime=1790121756791). The origin ticket is not created by me.

I don't think the sample will resolve this issue as it is asked to add a message by `aria-describedBy`.  Based on my understanding, we have a input control that needed to add a supplementary information rather than a label.
